src/tsdb.h

```diff
@@ -24,7 +25,8 @@ typedef struct CompactionRule {
 
 typedef struct Series {
     RedisModuleDict* chunks;
-    Chunk* lastChunk;
+//    Chunk* lastChunk;
```

Comment:
@ashtul you will have an isusue here because `Chunk*` should be some sort of a generic type, you can look at `redismodule.h` and see how redis declares a non-complete struct for example:
```
typedef struct RedisModuleDict RedisModuleDict;
```

Reply:
Will use Chunk_t which is `typedef void Chunk_t`